vm-validator/src/vm_validator.rs

```diff
@@ -67,11 +67,8 @@ impl TransactionValidation for VMValidator {
         let address = AccountAddress::new([0xff; ADDRESS_LENGTH]);
         let item = RequestItem::GetAccountState { address };
 
-        match self
-            .storage_read_client
-            .update_to_latest_ledger(/* client_known_version = */ 0, vec![item])
-        {
-            Ok((mut items, ledger_info_with_sigs, _, _)) => {
+        match self.storage_read_client.retrieve_items(vec![item]) {
+            Ok((mut items, ledger_info_with_sigs)) => {
```

Comment:
All this code does is that it gets the latest state root hash and version. I'll add an API to just do that.

Reply:
Note that the other place in vm validator does need the account state of the address (its seq number and balance)